app/components/special-coverage/special-coverage-edit/special-coverage-edit.html

```diff
@@ -107,9 +107,8 @@
         <label for="specialCoverageCampaign">Sponsor Campaign</label>
         <autocomplete-basic
             class="form-control"
-            ng-if="!model.campaign || model.campaign.$resolved"
```

Comment:
I think, because of the way `autocomplete-basic` works at the moment, there's no way around the flicker.